### ISSUE TITLE
Error on npm install

### ISSUE THREAD

**Issue by ferdicus** [transistorsoft/rn-background-geolocation-demo#43]
Getting this error when trying to run `npm install` 
```
error "///Users/chris/Documents/workspace/react/background-geolocation/react-native-background-geolocation" doesn't exist.
```

I'm guessing I'm supposed to download the package from the npm repo instead of your Users folder :)

**Comment by christocracy** [comment]
oops!

**Comment by christocracy** [comment]
Fixed in master -> `"^2.8.2"`

**Comment by ferdicus** [comment]
Happens to the best - thanks a lot :)
